tests/dummy/app/pods/demo/template.hbs

```diff
@@ -26,4 +26,12 @@ as |slot|}}
   {{#block-slot slot 'filters' as |filters onFilter|}}
     {{frost-object-browser-filter filters=filters onFilter=onFilter}}
   {{/block-slot}}
+  {{#block-slot slot 'titles' as |titles|}}
```

Comment:
Using the plural `titles` here sounds weird to me, I'd think `title` makes more sense.